conformance/lang/values-types-and-variables/nil-literal.balt

```diff
@@ -0,0 +1,173 @@
+Test-Case: output
+Description: Test nil type descriptor with nil literal '()'.
+Labels: nil-type-descriptor, nil-literal
+
+public function main() {
+    () x = ();
+    io:println(x.toBalString()); // @output ()
+}
+
+Test-Case: output
+Description: Test nil type descriptor with nil literal 'null'.
+Labels: nil-type-descriptor, nil-literal
+
+public function main() {
+    () x = null;
+    io:println(x.toBalString()); // @output ()
+}
+
```

Comment:
Shall we duplicate these two test cases, without using toBalString(). So the output will be just an empty string. This is because there are no test cases covering normal toString behavior.